tests/expressions/test_sum.py

```diff
@@ -13,3 +13,19 @@ def test_unique_variables_sum_one_variable():
 def test_unique_variables_sum_two_variables():
     exp = Sum(Variable("x"), Variable("y")).unique_variables()
     assert exp == {"x", "y"}
+
+def test_evaluate_sum_two_numbers():
+    exp = Sum(Number (20), Number (7)).evaluate()
+    assert exp == 27
+
+def test_evaluate_sum_three_numbers():
+    exp = Sum(Number (2), Number (2), Number (7)).evaluate()
+    assert exp == 11
+
+def test_evaluate_sum_ten_numbers():
+    exp = Sum(Number (3), Number (1), Number (4), Number (1), Number (5), Number (9), Number (2), Number (6), Number (5), Number (3)).evaluate()
+    assert exp == 39
+
+def test_equal_sum_two_variables():
+    exp = Sum(Variable("x"), Variable("y")).__eq__()
```

Comment:
I'm not sure why the test is passing, but this is how you would test that two sums are equal.
```suggestion
    assert  Sum(Variable("x"), Variable("y")) == Sum(Variable("y"), Variable("x"))
```